src/main/java/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,15 @@
+package service.impl;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileWriterImpl {
+    public void write(String filePath, String content) throws IOException {
```

Comment:
```suggestion
    public void write(String filePath, String content) {
```
Don’t  throw exception in the method signature